data/src/main/scala/tofu/data/Embedded.scala

```diff
@@ -110,8 +111,9 @@ trait ExceptTInstances1 {
 
     }
 
-  implicit def exceptTParallel[G[+_]: Parallel, E: Semigroup](implicit
-      G: Monad[G]
+  implicit def exceptTParallel[G[+_], E: Semigroup](implicit
+      G: Monad[G],
+      @unused P: Parallel[G]
```

Comment:
I suppose this instance is still needed as evidence but is not used directly? @Odomontois

Reply:
There was a bug. Fixed